lib/Compile.js

```diff
@@ -9,15 +9,32 @@ var Compile = module.exports = function (options) {
       
       var width = options.width;
       var jsonData = options.data;
+      
+      if(options.stickerSheet) {
+        var startLength = jsonData.length-1;
+        
+        for(var i=1; i<139; i++) {
```

Comment:
The other place 139 was being used was in an unused variable below, so I brought it up to this for loop and used (var i=1; i<bleedLayerLength; i++) instead